dspace-api/src/test/java/org/dspace/content/RelationshipServiceImplTest.java

```diff
@@ -110,10 +110,10 @@ public void testFindByItem() throws Exception {
         relationshipTest.add(getRelationship(cindy, hank, hasFather,0,0));
         relationshipTest.add(getRelationship(fred, cindy, hasMother,0,0));
         relationshipTest.add(getRelationship(bob, cindy, hasMother,1,0));
-        when(relationshipService.findByItem(context, cindy, -1, -1)).thenReturn(relationshipTest);
+        when(relationshipService.findByItem(context, cindy, -1, -1, false)).thenReturn(relationshipTest);
```

Comment:
I forgot to note this above, but we also should add some **new** tests to this PR to test the behavior when `excludeTilted=true`...namely we should test that that side of the relationship is actually excluded as expected.